src/screens/SurveyDetail/question.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import Rating from 'components/Rating';
+import NetPromotingScore from 'components/NetPromotingScore';
```

Comment:
@rafayet-monon  The same comment @Lahphim added on [this PR](https://github.com/rafayet-monon/surveyor/pull/20/files#r544779215) 🙌🏼